sql/npc_list.sql

```diff
@@ -6253,7 +6253,7 @@ INSERT INTO `npc_list` VALUES (17056402,'Wathdeeh','Wathdeeh',0,0.000,0.000,0.00
 INSERT INTO `npc_list` VALUES (17056403,'Wawaroon','Wawaroon',0,0.000,0.000,0.000,0,40,40,0,0,0,6,25,0x0000C70600000000000000000000000000000000,32,'TOAU',1);
 INSERT INTO `npc_list` VALUES (17056404,'qm9','???',130,-405.888,6.514,-440.237,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',0);
 INSERT INTO `npc_list` VALUES (17056405,'14','     ',0,0.000,0.000,0.000,0,40,40,0,0,0,2,2075,0x0100000000100020003000400050006000700000,32,'TOAU',1);
-INSERT INTO `npc_list` VALUES (17056406,'15','     ',225,342.129,36.509,-24.856,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
+INSERT INTO `npc_list` VALUES (17056406,'Blu_Af2','     ',225,342.129,36.509,-24.856,32769,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'TOAU',1);
```

Comment:
Is this the only thing this npc in the game does is deal with blu quest?

Reply:
As far as i know it is only used for BLU AF2 quest. It's a blank npc on the floor. I can change it back but just named it before as on other PRs i submitted before i was asked to name it something related to what it is used for if its a blank or duplicate. (was just trying to avoid having to have discussions about it lol but again no issues changing it back just need confirmation :) )

Reply:
Tagging @TeoTwawki  for his insight in naming unnamed npcs :cat:

Reply:
it has no other uses, name doesn't much matter since blank in game except for being readable/recognizable to us humans.

Reply:
> ships
coz moving zone parts.

also certain bcnm entrance (divine might..) and door etc, expect the name to match wtf they reference.

those are usually (nut not always named the hex value of whatever Nth position they exist in zone, prefixed with a symbol like: `_07f` or matching the fourcc of their accompanying animation loop for objects that just sit and sparkle but can't be clicked on (holiday decorations will animate automatically if correctly named).

/endClientTrivia

Reply:
changing it back to 15, until there's a confirmation that this blank npc is not used for something else (can't be proven as most of Aht Ughan quests/misisions are not scripted).

Change will be seen on the next push (in a few minutes)

Reply:
I'm positive that it doesn't have any missions involved nor any non blu quest.